core/controllers/base_test.py

```diff
@@ -73,11 +73,13 @@ def test_dev_indicator_appears_in_dev_and_not_in_production(self):
 
         with self.swap(feconf, 'DEV_MODE', True):
             response = self.testapp.get(feconf.LIBRARY_INDEX_URL)
-            self.assertIn('<div class="oppia-dev-mode">', response.body)
+            self.assertIn('DEV_MODE: JSON.parse(\'true\')',
```

Comment:
The test def name is different then what you are testing, I would suggest you check the indicator div.

Reply:
Can I change the scope of the test from finding indicator div to asserting DEV_MODE boolean variable is set correctly ( thus disabling the "Dev Mode" indicator div)?

Reply:
I am going to test for both the div and the JSON boolean value for DEV_MODE.